### ISSUE TITLE
[Feature Request] Light Mode

### ISSUE THREAD

**Issue by GenericUsernameIsNotAvailable** [futo-org/grayjay-android#1103]
The current theme for GrayJay is dark.
While this is great for saving device energy, a lighter theme may be more appealing for daytime viewing as well as attracting new users to the app.

**Comment by EncomOS12** [comment]
I agree. I've had the app installed for a couple of months. I check it now and then for a light mode, when I don't see it I just close the app and come back a few weeks later. I don't use dark mode apps. Hurts my eyes

**Comment by kryptobolt07** [comment]
> The current theme for GrayJay is dark. While this is great for saving device energy, a lighter theme may be more appealing for daytime viewing as well as attracting new users to the app.

especially for devices that don't come with OLEDs(realistically the largest subset),the dark mode just sucks.

**Comment by VoxelPrismatic** [comment]
I think implementing a standard UI library should solve all these issues.

**Comment by khagler** [comment]
I would also like a light theme. I use my devices outdoors and in rooms with good lighting, and the gray text on black background is almost unreadable.

**Comment by EncomOS12** [comment]
At this point I have given up hoping for it and uninstalled the app.

If they can't do light mode I would at least prefer a "dark" mode instead of the "blackout" mode it currently has. Something with dark blue and gray. With blackout apps I can see the letters when I blink.

**Comment by kryptobolt07** [comment]
> At this point I have given up hoping for it and uninstalled the app.
> 
> If they can't do light mode I would at least prefer a "dark" mode instead of the "blackout" mode it currently has. Something with dark blue and gray. With blackout apps I can see the letters when I blink.

I actually made a hacky light mode(It ain't perfect) by manually changing the color values of the various components, now I ain't a mobile developer but honestly i do think they can implement this faster than some other features.

**Comment by rubaboo** [comment]
+1. Combined with tinier than normal (as in all other apps that conform to the Android setting for scaling) text, the complete black mode is brutal on eyes.

**Comment by ianmcl** [comment]
Auto theme switch with system, please

**Comment by Greysys** [comment]
+1. Please implement automatic light/dark mode, it is the one thing stopping me from using Grayjay full time. Dark mode is horrible especially when working close to a window.

**Comment by markcst** [comment]
Hi! Still no system theme mode with a light mode available?

**Comment by magiruuvelvet** [comment]
I want to add an actual accessibility need for a light theme. I have astigmatism. I exclusively own and use devices with LCD screens with anti-glare coating in well-lit environments. The specific dark theme used in Grayjay is genuinely frustrating to use, and the text is very hard to read. The specific issue I have is low contrast perception due to LCD backlight interference (in well-lit ambient light conditions). You guys might want to read more about "light text bleeding" and "halation" effects, which aren't uncommon problems with dark themes on LCDs in bright environments, especially for people with astigmatism.

**Comment by kafene** [comment]
@magiruuvelvet I have the same problem with astigmatism. Just trying Grayjay for the first time and surprised there's no way to use a light theme or increase font size. The tiny gray text on black is in some cases impossible for me to read.

**Comment by GenericUsernameIsNotAvailable** [comment]
@Zvonimir-FUTO is there an update with this in mind soon?
I know you've been busy with desktop version and plugins etc.

**Comment by Zvonimir-FUTO** [comment]
> [@Zvonimir-FUTO](https://github.com/Zvonimir-FUTO) is there an update with this in mind soon? I know you've been busy with desktop version and plugins etc.

No, unfortunately not. We are very constrained with time and we mostly focus on fixing stuff so this ended up on a back burner.

**Comment by tealtxgr** [comment]
> > [@Zvonimir-FUTO](https://github.com/Zvonimir-FUTO) is there an update with this in mind soon? I know you've been busy with desktop version and plugins etc.
> 
> No, unfortunately not. We are very constrained with time and we mostly focus on fixing stuff so this ended up on a back burner.

I absolutely **knew** this was gonna be your response. And I totally understand. It seems like every week you guys have to put out a new YouTube fire. I do want you to know your teams app has gotten me to try and move past YouTube. Im starting to pay for creators on other platforms now where I can, like Patreon and stuff, because YouTube is fighting tooth and nail to be the only player in town. 

GrayJay is a great app and it's genuinely kinda fun to use. Hopefully other people can start deprioritizing YT and hopefully more creators will switch to less locked down platforms. ❤️❤️

**Comment by shapirus** [comment]
to put it simply, this is insane. the app is unusable for many just because of this.

**Comment by rubaboo** [comment]
I know nobody cares, but I'm about to quit over this. Eyes getting old. I am not keeping up closely but it seems all that is happening recently is just keeping plugins up to date with source changes, and occasional bug squashing.
